modules/exploits/multi/http/osCommerce.rb

```diff
@@ -0,0 +1,69 @@
+##
+# This module requires Metasploit: https://metasploit.com/download
+# Current source: https://github.com/rapid7/metasploit-framework
+##
+
+class MetasploitModule < Msf::Exploit::Remote
+  Rank = GoodRanking
+
+  include Msf::Exploit::Remote::HttpClient
+
+  def initialize(info = {})
+    super(update_info(info,
+      'Name'           => 'osCommerce 2.3.4.1 - Remote Code Execution',
```

Comment:
Generally module names do not contain the software version.
Suggest:
osCommerce Unauthenticated Remote Code Execution

Reply:
The module name usually also describes the vulnerable functionality, such as the name of the vulnerable component, filename, or parameter. This helps avoid confusion in the event that more than one exploit is present.

The module is located in `exploits/multi/http/`, so `Remote` is implied.

Consider:
* `osCommerce install.php Unauthenticated Code Execution`
* `osCommerce Installer Unauthenticated Code Execution`

Reply:
Done, thank you for the feedback.